packages/inscription-view/src/components/editors/part/Part.tsx

```diff
@@ -1,79 +1,39 @@
+import { Button, Flex } from '@axonivy/ui-components';
+import { useInscriptionTabState, type PartProps } from './usePart';
+
 import { useTranslation } from 'react-i18next';
+import { InscriptionTabs, type InscriptionTab } from '../../widgets/inscription-tabs/InscriptionTabs';
 
-const Control = ({ name, reset, control, ...props }: Pick<PartProps, 'name' | 'reset' | 'control'> & AccordionControlProps) => {
+export const Control = ({ name, reset, control, ...props }: Pick<PartProps, 'name' | 'reset' | 'control'> & { className: string }) => {
   const { t } = useTranslation();
-  if (reset.dirty || control) {
-    return (
-      <Flex direction='row' gap={2} {...props}>
-        {control}
-        {reset.dirty && (
-          <Button
-            icon={IvyIcons.Undo}
-            onClick={reset.action}
-            title={t('label.resetPart', { name })}
-            aria-label={t('label.resetPart', { name })}
-          />
-        )}
-      </Flex>
-    );
-  }
-  return null;
-};
 
-const State = ({ state }: Pick<PartProps, 'state'>) => (
-  <AccordionState
-    messages={state.validations.map(({ message, severity }) => ({
-      message,
-      variant: severity.toLocaleLowerCase() as Lowercase<Severity>
-    }))}
-    state={state.state}
-  />
-);
-
-const Part = ({ parts }: { parts: PartProps[] }) => {
-  const { value, updateValue } = useAccordionState(parts);
   return (
-    <Accordion type='single' collapsible value={value} onValueChange={updateValue}>
-      {parts.map(part => (
-        <PartItem {...part} key={part.name} />
-      ))}
-    </Accordion>
+    <Flex direction='row' gap={2} justifyContent='flex-end' {...props}>
+      {control}
+      <Button
+        icon={IvyIcons.Undo}
+        onClick={reset.action}
+        title={t('label.resetPart', { name })}
+        aria-label={t('label.resetPart', { name })}
+        disabled={!reset.dirty}
+      />
+    </Flex>
   );
 };
 
-const PartItem = (part: PartProps) => {
-  const { ref, isSticky } = useSticky();
-  return (
-    <AccordionItem value={part.name} data-sticky={isSticky ? 'true' : undefined}>
-      <AccordionTrigger ref={ref} control={props => <Control {...props} {...part} />} state={<State state={part.state} />}>
-        {part.name}
-      </AccordionTrigger>
-      <AccordionContent>
-        <ErrorBoundary FallbackComponent={ErrorFallback} resetKeys={[part]}>
-          <Flex direction='column' gap={3}>
-            {part.content}
-          </Flex>
-        </ErrorBoundary>
-      </AccordionContent>
-    </AccordionItem>
-  );
+const Part = ({ parts }: { parts: PartProps[] }) => {
+  const { value, updateValue } = useInscriptionTabState(parts);
+  const tabs: InscriptionTab[] = parts.map(part => ({
+    id: part.name,
+    name: part.name,
+    content: part.content,
+    reset: part.reset,
+    state: part.state,
+    icon: part.icon ? part.icon : IvyIcons.UserDialog,
+    control: part.control
+  }));
```

Comment:
not sure why so complicated...
```suggestion
  const tabs: InscriptionTab[] = parts.map(part => ({
    ...part,
    id: part.name
  }));
```

see also my other comment, maybe this translation is not needed at all.